zrml/court/src/lib.rs

```diff
@@ -1118,14 +1118,40 @@ mod pallet {
             Weight::zero()
         }
 
-        // Get `n` unique and ordered random numbers from the random number generator.
+        // The algorithm uses reservoir sampling (Algorithm R),
+        // which is an efficient technique for randomly choosing a sample of k elements
+        // from a larger population without replacement.
+        // Custom sampling function for u128
         // This follows the rule of draw without replacement.
+        fn get_n_random_indices(range: u128, count: usize) -> Vec<u128> {
+            use rand::Rng;
```

Comment:
Whenever I do an import directly at the point where I am using it, it is because of laziness to scroll to the top of the file and back. Although I know the keyboard shortcuts :D